app/styles/ui/_pull-request-diff.scss

```diff
@@ -1,4 +1,14 @@
-.pull-request-diff-viewer {
-  height: 500px;
-  display: flex;
+.pull-request-files-changed {
+  border: var(--base-border);
+  border-radius: var(--border-radius);
+
+  .files-changed-header {
+    padding: var(--spacing-half);
```

Comment:
Based on the [figma](https://www.figma.com/file/PUqR1Esg6fVZbg1sa8vsko/Pull-Request-Exploration?node-id=5540%3A22270), this heading was meant to be narrow I think to try not to take more vertical space than needed. Thus, I used ```--spacing-half```, but realized it makes the text in this header not line up with other text in the dialog.
![Screen Shot 2022-09-21 at 12 23 21 PM](https://user-images.githubusercontent.com/75402236/191558867-8c149437-31bd-4275-bc20-94ba9f862cc9.png)



Then I tried --spacing-half for just vertical padding and it feels a little off to me. Thus, I think I lean towards 
``` var(--spacing-half) var(--spacing) ```
![image](https://user-images.githubusercontent.com/75402236/191558289-2b34f2a3-2617-494a-937d-c773545bf7e4.png)


Thus, I think I like regular padding, but didn't know if it felt like too much to others?

```var(--spacing)```
![image](https://user-images.githubusercontent.com/75402236/191558043-9fd733dc-e85a-4895-9b35-cd3ba0e556bd.png)

Reply:
I think using the regular padding (final screenshot) makes the most sense. It better visually separates it as a heading rather than letting it blend in as a file.